twilio/Module.md

```diff
@@ -281,57 +139,22 @@ public function main() returns error?{
     } else {
         log:printInfo(details.message());
     }
-}
-
 ```   
-
-### Get a message
-This section shows you how to get a message details  from your account. you need to provide message sid to retreive the details from the message list of your account. If the request is successful, it will send the MessageResourceResponse record else an error message with the details.
+* Get a message
```

Comment:
Suggestion:
" - **Get a message**
     This section shows you how to get details of a message from your account. You need to provide the message SID (<EXPANSION_OF_SID>) to retrieve the details from the message list of your account. If the request is successful, you will receive the `MessageResourceResponse` record. If not, you will receive an error message with details."